lib/collection/src/shards/transfer/transfer_tasks_pool.rs

```diff
@@ -48,6 +48,18 @@ impl TransferTaskProgress {
             eta: EtaCalculator::new(),
         }
     }
+
+    pub fn add(&mut self, delta: usize) {
+        self.points_transferred += delta;
+        self.points_total = max(self.points_total, self.points_transferred);
```

Comment:
Why is `points_total` changing during the resharding?
Is to account for some ongoing updates?

Reply:
The total count is an estimate, as it's **very** expensive to do exact counting here.

It therefore is expected to overshoot the estimated total about 50% of the time.